dm1_registers.xml

```diff
@@ -264,20 +267,27 @@
         If \Fsberror isn't 0 then accesses return error, and don't do anything
         else.
 
-        If the bus master is busy then accesses set \Fsberror, return error,
+        Writes to this register:
+
+        1. If the bus master is busy then accesses set \Fsberror, return error,
```

Comment:
Good point. It ends up inside a table cell, which does seem to have a lot of limits on it. Quick Googling doesn't tell me.